bundle.yaml

```diff
@@ -6,10 +6,10 @@
 #
 series: xenial
 description: |
-  Production-grade Kubernetes cluster with logging and monitoring
+  Production-grade Canonical Kubernetes cluster featuring logging and monitoring
 tags:
-- conjure-up-kubernetes
```

Comment:
Was this removed on purpose? @battlemidget  do we need this this tag for a functional reason?